dot/network/warp_sync.go

```diff
@@ -8,18 +8,34 @@ import (
 	"fmt"
 
 	"github.com/ChainSafe/gossamer/dot/network/messages"
+	"github.com/ChainSafe/gossamer/internal/primitives/consensus/grandpa"
+	"github.com/ChainSafe/gossamer/internal/primitives/core/hash"
+	"github.com/ChainSafe/gossamer/internal/primitives/runtime"
+	"github.com/ChainSafe/gossamer/internal/primitives/runtime/generic"
 	"github.com/ChainSafe/gossamer/lib/common"
 	libp2pnetwork "github.com/libp2p/go-libp2p/core/network"
 	"github.com/libp2p/go-libp2p/core/peer"
 )
 
 const MaxAllowedSameRequestPerPeer = 5
 
+type WarpSyncVerificationResult struct {
+	SetId         grandpa.SetID
+	AuthorityList grandpa.AuthorityList
+	Header        generic.Header[uint64, hash.H256, runtime.BlakeTwo256]
```

Comment:
I think that the type for a block header number is uint (I know that for 64bit OS it is the same as uint64, but we want to standardise that for Gossamer)